Installing-Oppia-(Windows;-Python-3).md

```diff
@@ -204,7 +204,8 @@ Any VM manager is fine, but the instructions here are specific to VirtualBox.
 
 ## Install Ubuntu 18 ISO
 
-1. Download the Ubuntu 18.04 64bit ISO from [here](https://releases.ubuntu.com/18.04/).
+1. Download the Ubuntu 18.04 64bit ISO from [here](https://releases.ubuntu.com/18.04/) or download the the version 22.0.4 from [here] https://releases.ubuntu.com/jammy/ubuntu-22.04.3-desktop-amd64.iso .
+
```

Comment:
I think we didn't mention other versions, as we didn't test setup for them. Also as per line 80 - https://github.com/oppia/oppia-web-developer-docs/pull/291/files#diff-d454b4ebe056b258e24eab60f5603a8ab731ccd85b00cbfcafe4e15f780032c0R80. It will be a arduous task as it comes as a barebones config.

Reply:
Just to clarify,  I mentioned Ubuntu 22 because I was able to install , and run Oppia without any issue.   I am  also using the recommended configuration with virtualbox.

Reply:
Ah alright, can you also let us know if there was any step you had to follow during installation which is currently not mentioned in wiki ?